test/unit/accuracy.test.js

```diff
@@ -32,12 +38,30 @@ const resultsState = {
     }
   ],
   labelColumn: 'height',
-  labelColumnCategorical: 'sun',
   accuracyCheckPredictedLabels: [4.0, 3.75, 2.63, 2.46, 1.6, 1.0],
-  accuracyCheckLabels: [5.9, 3.8, 2.6, 2.5, 1.6, 0.7]
+  accuracyCheckLabels: [5.9, 3.8, 2.6, 2.5, 1.6, 0.7],
+  accuracyCheckExamples: [[2], [2], [1], [1], [0], [0]],
+  selectedFeatures: ['sun'],
+  columnsByDataType: columnsByDataType,
+  featureNumberKey: {
+    'sun': {
+      'low' : 0,
+      'medium' : 1,
+      'high' : 2,
+    }
+  }
```

Comment:
There's a lot of state to set for these tests, which I suspect will be reusable in other upcoming tests. How do folks feel if I extract test data into its own file to re-use?

Reply:
No objections to factoring as feels appropriate.